core-primitives/component-container/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "itp-component-container"
```

Comment:
New utility crate for handling our components, similar to a dependency injection framework. Although it's very far from that still.